tests/ops/op_math/test_prod.py

```diff
@@ -1184,11 +1168,11 @@ def test_sorting_operators_with_wire_map(self):
             qml.PauliX(5),
             qml.Toffoli([2, "three", 4]),
             qml.CNOT([2, 5]),
-            qml.RX("test", 5),
+            qml.RX(1, 5),
             qml.PauliY(0),
-            qml.CRX("test", [0, 2]),
+            qml.CRX(1, [0, 2]),
             qml.PauliZ("three"),
-            qml.CRY("test", ["test", 2]),
+            qml.CRY(1, ["test", 2]),
```

Comment:
Why did these tests need to get changed?

Reply:
I messed up when creating this tests. `"test"` corresponds to a wire value, but I added it to the rotation angle.